lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -69,7 +71,36 @@ bool inAttributeChangedNotificationGuard()
     return attributeChangedNotificationGuardCount.load() > 0;
 }
 
-std::unordered_map<Ufe::Path, std::string> pendingAttributeChangedNotifications;
+std::unordered_map<Ufe::Path, TfToken> pendingAttributeChangedNotifications;
+
+void sendValueChanged(const Ufe::Path& ufePath, const TfToken& changedToken)
+{
+    Ufe::AttributeValueChanged vc(ufePath, changedToken.GetString());
+    Ufe::Attributes::notify(vc);
+
+    if (changedToken == HdCameraTokens->horizontalAperture
```

Comment:
Style nit-pick: I would have made a static set of tokens and used find on that:
static std::unordered_set<TfToken, TfToken::HashFunctor> cameraTokens{HdCameraTokens->horizontalAperture, HdCameraTokens->verticalAperture, // ...  etc. };
if (cameraTokens.count(changedToken)) {
    Ufe::Camera::notify(ufePath);
}
Not sure which is faster --- unordered_set is amortized O(1), the chain of possibilities is O(n), but that's just complexity, not real-life performance.  Your pick.